src/datamigration/azext_datamigration/manual/custom.py

```diff
@@ -0,0 +1,238 @@
+                             overwrite=False,
+                             config_file_path=None):
+
+    try:
+
+        if not platform.system().__contains__('Windows'):
+            raise CLIError("This command cannot be run in non-windows environment. Please run this command in Windows environment")
+
+        defaultOutputFolder = get_default_output_folder()
+
+        # Assigning base folder path
+        baseFolder = os.path.join(defaultOutputFolder, "Downloads")
+        exePath = os.path.join(baseFolder, "SqlAssessment.Console.csproj", "SqlAssessment.exe")
+
+        # Creating base folder structure
+        if not os.path.exists(baseFolder):
+            os.makedirs(baseFolder)
+
+        testPath = os.path.exists(exePath)
+
+        # Downloading console app zip and extracting it
+        if not testPath:
+            zipSource = "https://sqlassess.blob.core.windows.net/app/SqlAssessment.zip"
+            zipDestination = os.path.join(baseFolder, "SqlAssessment.zip")
+
+            urllib.request.urlretrieve(zipSource, filename=zipDestination)
+            with ZipFile(zipDestination, 'r') as zipFile:
+                zipFile.extractall(path=baseFolder)
+
+        if connection_string is not None:
+            connection_string = ", ".join(f"\"{i}\"" for i in connection_string)
+            cmd = f'{exePath} Assess --sqlConnectionStrings {connection_string} ' if output_folder is None else f'{exePath} Assess --sqlConnectionStrings {connection_string} --outputFolder "{output_folder}" '
+            cmd += '--overwrite False' if overwrite is False else ''
+            subprocess.call(cmd, shell=False)
+        elif config_file_path is not None:
+            validate_config_file_path(config_file_path)
+            cmd = f'{exePath} --configFile "{config_file_path}"'
+            subprocess.call(cmd, shell=False)
+        else:
+            raise CLIError('No valid parameter set used. Please provide any one of the these prameters: connection_string, config_file_path')
+
+        # Printing log file path
+        logFilePath = os.path.join(defaultOutputFolder, "Logs")
+        print(f"Event and Error Logs Folder Path: {logFilePath}")
+
+    except Exception as e:
+        raise e
+
+
+# -----------------------------------------------------------------------------------------------------------------
+# Assessment helper function to test whether the given cofig_file_path is valid and has valid action specified.
+# -----------------------------------------------------------------------------------------------------------------
+def validate_config_file_path(path):
+
+    if not os.path.exists(path):
+        raise CLIError(f'Invalid config file path: {path}. Please provide a valid config file path.')
+
+    # JSON file
+    with open(path, "r", encoding=None) as f:
+        configJson = json.loads(f.read())
+    try:
+        if not ((configJson['action'] == "Assess") or (configJson['action'] == "assess")):
```

Comment:
Please make this comparison case insensitive to allow all combinations of Assess.